.lintstagedrc

```diff
@@ -0,0 +1,3 @@
+{
+  "*": "prettier --write -u"
```

Comment:
Shouldn't eslint be added here as well?

Reply:
I'll add it using `eslint --fix`. We'll see if that causes any problems.